src/Paramore.Brighter.Outbox.DynamoDB/DynamoDbOutbox.cs

```diff
@@ -266,12 +272,14 @@ public void MarkDispatched(Guid id, DateTime? dispatchedAt = null, Dictionary<st
          int pageNumber = 1, 
          Dictionary<string, object> args = null)
         {
+            var now = DateTime.UtcNow;
+            
             if (args == null)
             {
                 throw new ArgumentException("Missing required argument", nameof(args));
             }
-            
-            var sinceTime = DateTime.UtcNow.Subtract(TimeSpan.FromMilliseconds(millisecondsDispatchedSince));
+
+            var dispatchedTime = now.Subtract(TimeSpan.FromMilliseconds(millisecondsDispatchedSince));
```

Comment:
This should be Inserted/Created time, as if it has a dispatched time it should not be returned in this call as it is not longer outstanding